include/deal.II/grid/reference_cell.h

```diff
@@ -1390,6 +1767,70 @@ ReferenceCell::face_indices() const
 
 
 
+inline unsigned int
+ReferenceCell::n_isotropic_refinement_choices() const
+{
+  switch (this->kind)
+    {
+      case ReferenceCells::Vertex:
+        return 1;
+      case ReferenceCells::Line:
+        return 1;
+      case ReferenceCells::Triangle:
+        return 1;
+      case ReferenceCells::Quadrilateral:
+        return 1;
+      case ReferenceCells::Tetrahedron:
+        return 3;
+      case ReferenceCells::Pyramid:
+        return 1;
+      case ReferenceCells::Wedge:
+        return 1;
+      case ReferenceCells::Hexahedron:
+        return 1;
+      default:
+        DEAL_II_ASSERT_UNREACHABLE();
+    }
+
+  return numbers::invalid_unsigned_int;
+}
+
+
+
+inline std::vector<std::uint8_t>
+ReferenceCell::all_isotropic_refinement_choices() const
```

Comment:
Instead of returning a vector, could we pass in an index and only return a value?

Reply:
After the naming of the function, I think it should stay a vector, as it makes an easy iteration over all cases possible (yet I did not use it anywhere in the code so it is no problem for me to change it). If we want to change it maybe the name should be changed to `ReferenceCell::get_isotropic_refinement_choice()`?

Reply:
Renamed the function to indicate that now only a value is returned.